packages/edit-post/src/components/header/index.js

```diff
@@ -38,7 +39,10 @@ function Header( {
 			className="edit-post-header"
 			tabIndex="-1"
 		>
-			<HeaderToolbar />
+			<div>
```

Comment:
@kjellr or @jasmussen - I don't know if this is the best idea to wrap them with div but I took as little steps as possible to make it look close to what we had before. I would love to hear your recommendations because I also noticed that the border is slightly broken now:

![Screen Shot 2019-05-17 at 10 02 08](https://user-images.githubusercontent.com/699132/57912539-da9c3100-788a-11e9-83b1-31295ac4608e.png)

Reply:
Yep was about to note the same. There's a height difference to the toolbar. Even outside of fullscreen. This branch:

![Screenshot 2019-05-17 at 10 58 51](https://user-images.githubusercontent.com/1204802/57916388-0d4a2780-7893-11e9-9285-670c86ece0f6.png)

Master:

![Screenshot 2019-05-17 at 10 59 47](https://user-images.githubusercontent.com/1204802/57916404-11764500-7893-11e9-8c78-9c1d2f035049.png)

Let me take a look.

Reply:
It looks like if this particular div is assigned `display: flex;`, and `.edit-post-fullscreen-mode-close__toolbar` is also assigned `display: flex;` (instead of `inline-block` as it is now), both issues are fixed:

![Screenshot 2019-05-17 at 11 03 19](https://user-images.githubusercontent.com/1204802/57916616-7f227100-7893-11e9-89ba-3aae74a90a23.png)

![Screenshot 2019-05-17 at 11 03 38](https://user-images.githubusercontent.com/1204802/57916625-834e8e80-7893-11e9-80a7-2f0a2f5e7125.png)

I could use a sanity check by Kjell though.

Reply:
> It looks like if this particular div is assigned display: flex;, and .edit-post-fullscreen-mode-close__toolbar is also assigned display: flex; (instead of inline-block as it is now), both issues are fixed:

Yep, that fixes it for me too. 👍